rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -515,7 +498,7 @@
                 <javaclass references="org.hibernate.metamodel.spi.{type}"/>
             </when>
             <perform>
-                <hint title="org.hibernate.metamodel.spi.{type} was moved in Hibernate 5" effort="1" category-id="mandatory">
+                <hint title="Hibernate 5 - Moved method org.hibernate.metamodel.spi.{type}" effort="1" category-id="optional">
```

Comment:
According to Marc, this is optional because the common practice is to disable Hibernate 5 and add legacy Hibernate. This should also be explained in the hint.